yarn.lock

```diff
@@ -303,9 +303,9 @@
     jest "^26.5.2"
 
 "@glennsl/bs-json@^5.0.0":
-  version "5.0.2"
-  resolved "https://registry.yarnpkg.com/@glennsl/bs-json/-/bs-json-5.0.2.tgz#cfb85d94d370ec6dc17849e0ddb1a51eee08cfcc"
-  integrity sha512-vVlHJNrhmwvhyea14YiV4L5pDLjqw1edE3GzvMxlbPPQZVhzgO3sTWrUxCpQd2gV+CkMfk4FHBYunx9nWtBoDg==
+  version "5.0.4"
```

Comment:
This broke some jest tests that relied on some errors that were improved in bs-json, could the snapshots be updated please?

Reply:
I'm sorry but they aren't failing for me. Am I doing something wrong?